cmd/neofs-node/cache.go

```diff
@@ -227,3 +229,49 @@ func (s *lruNetmapSource) getNetMapByEpoch(epoch uint64) (*netmapSDK.Netmap, err
 func (s *lruNetmapSource) Epoch() (uint64, error) {
 	return s.netState.CurrentEpoch(), nil
 }
+
+// wrapper over TTL cache of values read from the network
+// that implements container lister.
+type ttlContainerLister ttlNetCache
+
+func newCachedContainerLister(w *wrapper.Wrapper) *ttlContainerLister {
+	const (
+		containerListerCacheSize = 100
+		containerListerCacheTTL  = 30 * time.Second
+	)
+
+	lruCnrListerCache := newNetworkTTLCache(containerListerCacheSize, containerListerCacheTTL, func(key interface{}) (interface{}, error) {
+		var (
+			id    *owner.ID
+			strID = key.(string)
+		)
+
+		if strID != "" {
+			id := owner.NewID()
```

Comment:
shouldn't the id here be the same as in `var` section?

Reply:
Fixed